test/00/equationDisplay.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/sh
+
+here=`pwd`
+if test $? -ne 0; then exit 2; fi
+tmp=/tmp/$$
+mkdir $tmp
+if test $? -ne 0; then exit 2; fi
+cd $tmp
+if test $? -ne 0; then exit 2; fi
+
+fail()
+{
+    echo "FAILED" 1>&2
+    cd $here
+    chmod -R u+w $tmp
+    rm -rf $tmp
+    exit 1
+}
+
+pass()
+{
+    echo "PASSED" 1>&2
+    cd $here
+    chmod -R u+w $tmp
+    rm -rf $tmp
+    exit 0
+}
+
+trap "fail" 1 2 3 15
+
+# insert ecolab script code here
+# use \$ in place of $ to refer to variable contents
+# exit 0 to indicate pass, and exit 1 to indicate failure
+cat >input.tcl <<EOF
+source assert.tcl
+proc afterMinskyStarted {} {
+minsky.load $here/test/testEq.mky
+.tabs select 1
+minsky.equationDisplay.requestRedraw
+assert {[lindex [.tabs tabs] [.tabs index current]]==".equations"}
+#event generate .equations.canvas  <B1-Motion> -x 100 -y 100 -warp 1
+minsky.equationDisplay.requestRedraw
```

Comment:
Is this enough to trigger the redraw? Or do you need to bring the equation tab to the fore.

Reply:
I trigger redraw explicitly now, so let's hope it improves the coverage score. If it does, I'll add similar tests for the parameter and variable sheets...

Reply:
Hmmm, it seems like "if (surface.get())" in draw() is always false in parVarSheet.cc and minsky.cc, because the if conditional is touched but then skipped...